DependencyInjection/FOSOAuthServerExtension.php

```diff
@@ -78,6 +78,20 @@ public function load(array $configs, ContainerBuilder $container)
             }
         }
 
+        // Handle the MongoDB document manager name in a specific way as it does not have a registry to make it easy
+        // TODO: change it when bumping the requirement to Symfony 2.1
```

Comment:
I believe these requirements are already bumped. If this PR is to be merged to `master`, it should work with Symfony 3.0 onwards.